CHANGES.md

```diff
@@ -1,6 +1,361 @@
+* added support for comparing `c_fn_ptr` values against `nil`
+* added unstable methods to append numeric values to a `bytes`/`string`  
+  (see https://chapel-lang.org/docs/main/language/spec/bytes.html#Bytes.bytes.appendByteValues  
+   and https://chapel-lang.org/docs/main/language/spec/strings.html#String.string.appendCodepointValues)
+* added an unstable method to compute the hexadecimal encoding of a `bytes`  
+  (see https://chapel-lang.org/docs/main/language/spec/bytes.html#Bytes.bytes.toHexadecimal)
+
+New Package Module Features
+---------------------------
+* improved the `DistributedBag` package w.r.t. depth-first-search and quality  
+  (see https://chapel-lang.org/docs/main/modules/packages/DistributedBag.html)
+* significantly improved `Sort.sort()` performance for large problem sizes
+* improved the performance of `toHex()` routines in the 'Crypto' module
+
+Standard Domain Maps (Layouts and Distributions)
+------------------------------------------------
+
+Changes / Feature Improvements in Libraries
+-------------------------------------------
+* added the missing deserializer for the `bigint` type  
+  (see https://chapel-lang.org/docs/2.0/modules/standard/BigInteger.html#BigInteger.bigint.deserialize)
+
+Name Changes in Libraries
+-------------------------
+
+Deprecated / Unstable / Removed Library Features
+------------------------------------------------
+* unstabilized `imag` overloads of trigonometric/hyperbolic 'Math' functions
+* removed the remaining deprecated routines/constants from the 'Math' module  
+  (e.g., `carg()`, `conjg()`, `divceil()`, `log2_e`, `half_pi`, etc.)
+* removed deprecated 'IO' support for the `readThis()`/`writeThis()` methods
+* removed the deprecated 'IO' configs that control `region` argument behavior  
+  (e.g. `useNewSeekRegionBounds`, `useNewOpenReaderRegionBounds`, etc.)
+* removed all previously deprecated `BigInteger` functions
+* removed the deprecated 'BigInteger' config `bigintInitThrows`
+* removed some of the previously deprecated `Time.date[time]` methods
+* removed the previously deprecated `Time.getCurrentTime()` function
+* removed some of the previously deprecated `Time` types
+  (e.g., `datetime`, `timedelta`, `Timer`, `TZInfo`, `TimeUnits`, etc.)
+* removed all previously deprecated `FileSystem` functions
+* removed the previously deprecated `BitOps.popcount()` function
+
+GPU Computing
+-------------
+* improved GPU sorting routines w.r.t. performance and supported types  
+  (see https://chapel-lang.org/docs/2.0/modules/standard/GPU.html#GPU.gpuSort)
+* enabled using ROCm's LLVM using `CHPL_LLVM=system` and `CHPL_GPU=amd`
+  (see TODO)
+* eliminated unnecessary synchronization for reductions on AMD GPUs
+
+Performance Optimizations / Improvements
+----------------------------------------
+* significantly improved `Sort.sort()` performance for large problem sizes
```

Comment:
Noting that this is listed here as well as in `New Package Module Features`; probably just one place would do